br-ext/package/optee_os_services/optee_os_services.mk

```diff
@@ -0,0 +1,38 @@
+# This recipe build the services TAs from optee_os and installs them in the
+# root file system at path /lib/optee_armtz/.
+#
+# BR2_PACKAGE_OPTEE_OS_SERVICES_SITE sets the optee_os source file tree path.
+# BR2_PACKAGE_OPTEE_OS_SERVICES_SDK sets TA devkit path.
+# BR2_PACKAGE_OPTEE_OS_SERVICES_CROSS_COMPILE sets cross compiler path.
+
+OPTEE_OS_SERVICES_VERSION = 1.0
+OPTEE_OS_SERVICES_SOURCE = local
+OPTEE_OS_SERVICES_SITE = $(BR2_PACKAGE_OPTEE_OS_SERVICES_SITE)
+OPTEE_OS_SERVICES_SITE_METHOD = local
+OPTEE_OS_SERVICES_INSTALL_STAGING = YES
+
+OPTEE_OS_SERVICES_LICENSE = BSD-2-Clause
+OPTEE_OS_SERVICES_LICENSE_FILES = LICENSE
+
+define OPTEE_OS_SERVICES_BUILD_CMDS
+	@$(foreach f,$(wildcard $(@D)/ta_services/*/Makefile), \
```

Comment:
I think  the last we said wasn't to use `ta_services` and instead just use `ta`. I.e., this should be updated accordingly.

Reply:
ok, fine. let's move service TAs into optee_os/ta/...